configs/14.0/release_notes/relfixes.html

```diff
@@ -11,6 +11,72 @@ <h3>Advance Toolchain new location</h3>
 <p>Since the end of August 2019, Advance Toolchain has been available from a new FTP server: <a href="ftp://public.dhe.ibm.com/">public.dhe.ibm.com</a></p>
 <p>Please be advised the FTP server at Unicamp won't be updated any longer.</p>
 
+<h3>New features in 14.0-0</h3>
+<ul>
+  <li>Support for Ubuntu 20.04.</li>
+  <li>GCC provides:</li>
+  <ul>
+    <li>Version 10.2, see the <a href="https://gcc.gnu.org/gcc-10/changes.html">release notes</a> for details.</li>
+    <li>MMA support.</li>
+  </ul>
+  <li>GNU C Library provides:</li>
+  <ul>
+    <li>Version 2.32, see the <a href="https://sourceware.org/pipermail/libc-announce/2020/000029.html">release notes</a> for details.</li>
+  </ul>
+  <li>Binutils provides:</li>
+  <ul>
+    <li>Version 2.35, see the <a href="https://lists.gnu.org/archive/html/info-gnu/2020-07/msg00008.html">release notes</a> for details.</li>
+  </ul>
+  <li>GNU Debugger provides:</li>
+  <ul>
+    <li>Version 9.2, see the <a href="https://lists.gnu.org/archive/html/info-gnu/2020-05/msg00008.html">release notes</a> for details.</li>
+  </ul>
+  <li>Userspace RCU provides:</li>
+  <ul>
+    <li>Version 0.12.1, see the <a href="https://github.com/urcu/userspace-rcu/blob/v0.12.1/ChangeLog">release notes</a> for details.</li>
+  </ul>
+  <li>OpenSSL provides:</li>
+  <ul>
+    <li>Version 1.1.1g, see the <a href="https://www.openssl.org/news/openssl-1.1.1-notes.html">release notes</a> for details.</li>
+  </ul>
+  <li>Expat provides:</li>
+  <ul>
+    <li>Version 2.2.9, see the <a href="https://github.com/libexpat/libexpat/blob/R_2_2_9/expat/Changes">release notes</a> for details.</li>
+  </ul>
+  <li>Libhugetlbfs provides:</li>
+  <ul>
+    <li>Version 2.21, see the <a href="https://github.com/libhugetlbfs/libhugetlbfs/blob/2.22/NEWS">release notes</a> for details.</li>
+  </ul>
+  <li>Python provides:</li>
+  <ul>
+    <li>Version 3.8.5, see the <a href="https://www.python.org/downloads/release/python-385/">release notes</a> for details.</li>
+  </ul>
+  <li>Boost provides:</li>
+  <ul>
+    <li>Version 1.73.0, see the <a href="https://www.boost.org/users/history/version_1_73_0.html">release notes</a> for details.</li>
+  </ul>
+  <li>libdfp provides:</li>
+  <ul>
+    <li>Version 0.15.0, see the <a href="https://github.com/libdfp/libdfp/releases/tag/1.0.15">release notes</a> for details.</li>
+  </ul>
+  <li>Valgrind provides:</li>
+  <ul>
+    <li>Version 3.15.0, see the <a href="http://valgrind.org/docs/manual/dist.news.html">release notes</a> for details.</li>
```

Comment:
Valgrind 3.16.1 has already been released, but a tag wasn't created.
We may want to update this later.

Reply:
I missed the release end of June.
I agree to update Valgrind in AT 14.0-1.